src/js/media-container.js

```diff
@@ -304,51 +304,66 @@ class MediaContainer extends window.HTMLElement {
     // This allows things like autoplay and programmatic playing to also initiate hiding controls (CJP)
     this.setAttribute('user-inactive', 'user-inactive');
 
-    const scheduleInactive = () => {
+    const setInactive = () => {
+      if (this.autohide < 0) return;
+      this.setAttribute('user-inactive', 'user-inactive');
+      const evt = new window.CustomEvent(
+        MediaStateChangeEvents.USER_INACTIVE,
+        { composed: true, bubbles: true, detail: true }
+      );
+      this.dispatchEvent(evt);
+    };
+
+    const setActive = () => {
       this.removeAttribute('user-inactive');
       const evt = new window.CustomEvent(
-        MediaStateChangeEvents.USER_INACTIVE, 
+        MediaStateChangeEvents.USER_INACTIVE,
         { composed: true, bubbles: true, detail: false }
       );
       this.dispatchEvent(evt);
+    }
+
+    const autohide = () => {
       window.clearTimeout(this._inactiveTimeout);
 
       // Setting autohide to -1 turns off autohide
       if (this.autohide < 0) return;
 
       this._inactiveTimeout = window.setTimeout(() => {
-        this.setAttribute('user-inactive', 'user-inactive');
-        const evt = new window.CustomEvent(
-          MediaStateChangeEvents.USER_INACTIVE, 
-          { composed: true, bubbles: true, detail: true }
-        );
-        this.dispatchEvent(evt);
+        setInactive();
       }, this.autohide * 1000);
+    }
+
+    const scheduleInactive = () => {
+      setActive();
```

Comment:
`scheduleInactive` was an existing method that when an event happened like a mousemove, keyup, or touch would make the controls active and then schedule hiding the controls on the timer.
I moved setActive into a separate function because there were two places where the same code was being called.